chromium_src/chrome/install_static/chromium_install_modes.cc

```diff
@@ -124,21 +124,21 @@ const InstallConstants kInstallModes[] = {
         sizeof(kInstallModes[0]),
         CANARY_INDEX,  // The mode for the side-by-side canary channel.
```

Comment:
also change this to `NIGHTLY_INDEX` along with the header and tests?

Reply:
Done.